packages/rsbuild/package.json

```diff
@@ -33,10 +33,32 @@
     "@nx/devkit": "file:../devkit",
     "@nx/js": "file:../js",
     "@rsbuild/core": "1.1.8",
-    "minimatch": "9.0.3"
+    "minimatch": "9.0.3",
+    "@phenomnomnominal/tsquery": "~5.0.1"
   },
   "peerDependencies": {},
   "nx-migrations": {
     "migrations": "./migrations.json"
+  },
+  "exports": {
+    ".": {
+      "types": "./index.d.ts",
+      "default": "./index.js"
+    },
+    "./package.json": {
+      "default": "./package.json"
+    },
+    "./generators": {
+      "types": "./generators.d.ts",
+      "default": "./generators.js"
+    },
+    "./config-utils": {
+      "types": "./config-utils.d.ts",
+      "default": "./config-utils.js"
+    },
+    "./plugin": {
```

Comment:
We can use `@nx/rsbuild` for the plugin rather than `@nx/rsbuild/plugin`. Thee other plugins couldn't do this because the `.` export was already used for other things.

Reply:
Let's leave this as is in this PR and follow up the change in a separate PR as there will be more things touched by the change from `@nx/rsbuild/plugin` to `@nx/rsbuild` 
(checking if build plugin exists, adding the plugin to nx.json, init, etc)